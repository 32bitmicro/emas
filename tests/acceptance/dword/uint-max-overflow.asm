.dword 4294967296
