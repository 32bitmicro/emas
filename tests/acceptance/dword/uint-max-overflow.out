acceptance/dword/uint-max-overflow.asm:1:18: Value won't fit in a DWORD: 4294967296
